data/RedHat.yaml

```diff
@@ -1,5 +1,6 @@
 ---
-tftp::service: tftp.socket
-tftp::package: tftp-server
-tftp::root: "/var/lib/tftpboot"
-tftp::syslinux_package: syslinux
+tftp::service: ['tftp.socket','tftp.service']
```

Comment:
I'm more fan of the multi line style.

But this does change some things: previously the service was only started when needed, now it is always enabled. I'm not sure what's better.

I guess you did this so you could reliably restart the service if an override changed, correct?

Reply:
> I guess you did this so you could reliably restart the service if an override changed, correct?

exactly

Reply:
and added multi line style